main/provider/chains/index.ts

```diff
@@ -86,6 +84,7 @@ function getActiveChains(): RPC.GetEthereumChains.Chain[] {
         chainId: id,
         networkId: id,
         name,
+        connected: chain.connection.primary.connected || chain.connection.secondary.connected,
```

Comment:
I don't think we need to add this property, rather clients can just send messages and they will fail if the connection is not active